src/common/Types.ts

```diff
@@ -362,6 +362,7 @@ export interface Stateless {}
 export interface CommonAccessibilityProps {
     // iOS, Android, and Desktop
     importantForAccessibility?: ImportantForAccessibility;
+    accessibilityId?: string;
```

Comment:
This new prop should be added to the documentation for all applicable components.